pkg/virt-operator/creation/components/crds.go

```diff
@@ -95,6 +95,7 @@ func NewVirtualMachineCrd() *extv1beta1.CustomResourceDefinition {
 			{Name: "Age", Type: "date", JSONPath: ".metadata.creationTimestamp"},
 			{Name: "Running", Type: "boolean", JSONPath: ".spec.running"},
 			{Name: "Volume", Description: "Primary Volume", Type: "string", JSONPath: ".spec.volumes[0].name"},
+			{Name: "Created", Type: "boolean", JSONPath: ".status.created", Priority: 1},
```

Comment:
@vatsalparekh could you add an an example to the PR description, which shows how it looks now?

Reply:
updated the description